docs/getting-started/os-support.rst

```diff
@@ -1,13 +1,36 @@
 OS Support by Linux Distributions and Version
 ==============================================
 
-The following matrix shows which Linux distributions, containers, and images are supported with which versions of ScyllaDB.
+The following matrix shows which Linux distributions, containers, and images
+are :ref:`supported <os-support-definition>` with which versions of ScyllaDB.
 
-Where *supported* in this scope means:
++-------------------------------+--------------------+-------+------------------+---------------+
+| Linux Distributions           |Ubuntu              | Debian| Rocky / Centos / | Amazon Linux  |
+|                               |                    |       | RHEL             |               |
++-------------------------------+------+------+------+-------+-------+----------+---------------+
+| ScyllaDB Version / OS Version |20.04 |22.04 |24.04 |  11   |   8   |   9      | 2023          |
++===============================+======+======+======+=======+=======+==========+===============+
+|   2025.1                      | |v|  | |v|  | |v|  | |v|   | |v|   | |v|      | |v|           |
++-------------------------------+------+------+------+-------+-------+----------+---------------+
+|   2024.2                      | |v|  | |v|  | |v|  | |v|   | |v|   | |v|      | |v|           |
++-------------------------------+------+------+------+-------+-------+----------+---------------+
+|   2024.1                      | |v|  | |v|  | |x|  | |v|   | |v|   | |v|      | |x|           |
++-------------------------------+------+------+------+-------+-------+----------+---------------+
+|   6.2                         | |v|  | |v|  | |v|  | |v|   | |v|   | |v|      | |v|           |
++-------------------------------+------+------+------+-------+-------+----------+---------------+
```

Comment:
Lets add "Enterprise" before 2024/5 and "Open Source" before 6.2

Reply:
2025.1 isn't named ScyllaDB Enterprise, just ScyllaDB.

Our documentation is versioned. Why does the documentation for 2025.1 has to document 2024.1? They have their own documentation pages.

Reply:
@tzach - please see above.